crates/bevy_input/src/axis.rs

```diff
@@ -2,8 +2,12 @@ use bevy_utils::HashMap;
 use std::hash::Hash;
 
 #[derive(Debug)]
+/// An axis that stores f32 values in range from `min` to `max`.
```

Comment:
Suggested alternative docs:

```rust
/// Stores the position data of input devices of type T
///
/// Values are stored as `f32` values, which range from `min` to `max`.
/// The default valid range is from -1.0 to 1.0, inclusive.
```
Reasons I like this framing:
- The first line in a doc comment in Rust should be useful in a standalone form, due to how they're used by IDEs in hover tooltiops
- We should explain the logical meaning (stores position data)
- We should explain how the type T is used
- We should mention that multiple axes can be stored in this single type.